### PR TITLE
hide bounties value when lower than $1

### PR DESCRIPTION
**Pull request by danlipert**
##### Description

This PR hides the token amount and USDT value when the USDT value is less than $1. This is to hide values for bounties where a placeholder amount of DAI is used to create the bounty, usually on behalf of another organization as part of a managed bounty. This request came from the hackathon support issues.

##### Refers/Fixes

https://github.com/gitcoinco/web/issues/4245

##### Testing

Tested locally - see video:
https://embed.vidyard.com/share/YoT3rDwFwQociVtoUD829L?

### CODE REVIEW

File: app/assets/v2/js/pages/bounty_details.js [hunk 1]
```diff
@@ -905,6 +905,10 @@ var build_detail_page = function(result) {
   });
 
   $('#bounty_details #issue_description code').parent().addClass('code-snippet');
+  if (result['value_in_usdt_now'] != 'undefined' && result['value_in_usdt_now'] <= 1) {
+    $('#value_in_usdt_wrapper').hide();
+    $('.token').hide();
+  }
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/bounty_details.js]: Do we need this ? 
we could do these checks within results.html itself using template `if`

Comment (thelostone-mc) [reply app/assets/v2/js/pages/bounty_details.js]: @danlipert ^ in case you didn't see it

Comment (octavioamu) [reply app/assets/v2/js/pages/bounty_details.js]: @danlipert this can be removed the changes in the template are doing this check already

**Comment by octavioamu** [comment]
@danlipert why this one was closed? not good?

**Comment by danlipert** [comment]
@octavioamu I thought you were saying this was already done at the template level in another PR :)

**Comment by octavioamu** [comment]
> @octavioamu I thought you were saying this was already done at the template level in another PR :)

no no no, I mean in my review what I was trying to say is your are using the conditional if in the jsrender template if > 1  so that will be processed there so you don't need to do it also in the js file

File: app/assets/v2/js/pages/bounty_details.js (hunk 1 above)
Comment (thelostone-mc) [reply app/assets/v2/js/pages/bounty_details.js]: Ideally, this should be done only in live ! we might need to revisit it

**Comment by danlipert** [comment]
Yeah I think this one is dead @thelostone-mc

**Pull request closed without merging**